docs/usage.md

```diff
@@ -47,7 +47,7 @@ You can avoid running any startup scripts by adding `class="mejs__player"` to th
 ### Vanilla JavaScript
 ```html
 <script>
-	var player = new MediaElementPlayer('player', {
+	var player = new MediaElementPlayer(document.querySelector('#mediaplayer'), {
```

Comment:
Yes. If you read the note I just setup, you can only use string for an ID, not for selectors. You will use `document.querySelector` for the rest of the situations